website/playground/Playground.js

```diff
@@ -368,6 +348,40 @@ class Playground extends React.Component {
       </EditorState>
     );
   }
+
+  setContent(content) {
+    this.setState({ content });
+  }
+
+  clearContent() {
+    this.setContent("");
+  }
```

Comment:
It'd be good to enable class fields in the Babel config and replace these with arrow functions assigned to fields, but I remember I had trouble with configuring that.